### ISSUE TITLE
Lore conflict in timeline

### ISSUE THREAD

**Issue by Konrad22** [wesnoth/wesnoth#8363]
### Source

Wesnoth Timeline

### Original Text

Official Wesnoth Timeline
1YW
[The Ruby of Fire is temporarily hidden, and the lich-lord Jevyan is deceived into believing it is held by the Elves.](https://wiki.wesnoth.org/Timeline_of_Wesnoth#25-40_YW)
[In **25 YW** Haldric II sends an expedition to retrieve the Ruby of Fire from its place of concealment.](https://wiki.wesnoth.org/Timeline_of_Wesnoth#25-40_YW)

Campaigns
Sceptre of Fire 
https://github.com/wesnoth/wesnoth/blob/60ae0db6118bdcda29ccf7f091084dc3a8afaeb9/data/campaigns/Sceptre_of_Fire/scenarios/1_A_Bargain_is_Struck.cfg#L336
-> meaning, Haldric I must have had the Ruby during Haldric II lifetime -> the Ruby should have been retrieved sometime earlier

Legend of Wesmere
https://github.com/wesnoth/wesnoth/blob/60ae0db6118bdcda29ccf7f091084dc3a8afaeb9/data/campaigns/Legend_of_Wesmere/scenarios/chapter3/12_Revelations.cfg#L87
https://github.com/wesnoth/wesnoth/blob/60ae0db6118bdcda29ccf7f091084dc3a8afaeb9/data/campaigns/Legend_of_Wesmere/scenarios/chapter3/12_Revelations.cfg#L97
-> claims explicitly that Haldric I has the Ruby already, but this talk happens in 20 YW

### Proposed Change

1) Rewrite all the references in all the campaigns to align with the timeline. XD
2) Edit the timeline.
2a) Say that Haldric I retrieved the Ruby sometime earlier, remove the other retrieval.
2b) Same as 2a), but make Haldric II store the Ruby away when Kalenz warns him about the effects of the Ruby, that way there is no need ro remove the mention of Haldric II recovering the Ruby.
3) Edit the lines in the campaign to make them compare the stories about Haldric I before the Ruby and after, instead of them saying that he holds the Ruby and grew worse because of it. Or maybe suggest that the exposion to the Ruby was enough to make him worsen over time either way.
4) Other suggestions, you name it. :)

In conclusion, as it is, the Official Timeline contradicts the campaigns.

**Comment by nemaara** [comment]
I'd say this is more of an issue with the timeline unless somewhere else in mainline mentions retrieving the Ruby? Which I somehow don't remember right now.

**Comment by Konrad22** [comment]
> I'd say this is more of an issue with the timeline unless somewhere else in mainline mentions retrieving the Ruby? Which I somehow don't remember right now.

Same, I can't remember any such mention, but I'm not 100% sure that there is none. :D

**Comment by stevecotton** [comment]
I can't remember any mention, but similarly can't be 100% sure.

Looking through the history of the Wiki page, the mention of retrieving the ruby was added in one of the many edits on 10th Oct 2008 ["Major revision folding in LoW material."](https://wiki.wesnoth.org/index.php?title=Timeline_of_Wesnoth&diff=27061&oldid=27059). Neither of the alternate histories merged on that day seems to mention retrieving it.

That coincides with LoW being added to mainline. It was added on 7th Oct, I haven't read the text as added, but looking though changes from (the commit after the big addition) through to the end of October, there seems to be no addition of retrieving the sceptre after the Wiki changes.

**Comment by Jonathan-Kelly** [comment]
Maybe just remove that one line explicitly mentioning the Ruby's retrieval from the timeline, since this is already implied by an earlier line at 1 YW:

`The Ruby of Fire is temporarily hidden, and the lich-lord Jevyan is deceived into believing it is held by the Elves.`

Here "temporarily hidden" indicates the Ruby will be retrieved, but without pinning its retrieval to a specific date.

**Comment by stevecotton** [comment]
While we're looking at that page, it would be nice if someone updated it to our current style guide, with [lower-case race names](https://wiki.wesnoth.org/Typography_Style_Guide#Race_Names).

**Comment by Jonathan-Kelly** [comment]
> it would be nice if someone updated it to our current style guide, with [lower-case race names](https://wiki.wesnoth.org/Typography_Style_Guide#Race_Names).

Done.
